.github/workflows/issues-stale.yml

```diff
@@ -1,7 +1,7 @@
 name: Close inactive issues
```

Comment:
I think we can just delete this

Reply:
I don't - it is a good way for us to catch inactive issues and follow them up.

Reply:
Lets look at the data. Can you point me to at least 3x issues where this was helpful and where it created meaningful follow-up conversations?

Reply:
There are no Stale issues to point you too.

Reply:
It has definitely prompted me a couple of times to take another look. Not sure how worthwhile it will be now that feature requests are closed, but I don't feel like it's hurting anything.

Reply:
The less stuff we have to maintain the better for everyone.

And if nobody can point me to specific instances where this has been helpful (not just in a "oh that's interesting" way, but actually helpful) - I would suggest to keep our scope to a minimum.

Reply:
Removed.